apps/emqx/src/emqx.appup.src

```diff
@@ -5,11 +5,19 @@
     [{load_module,emqx_channel,brutal_purge,soft_purge,[]},
      {load_module,emqx_schema,brutal_purge,soft_purge,[]},
      {load_module,emqx_release,brutal_purge,soft_purge,[]},
+     {load_module,emqx_broker,brutal_purge,soft_purge,[]},
+     {load_module,emqx_mqtt_caps,brutal_purge,soft_purge,[]},
+     {load_module,emqx_topic,brutal_purge,soft_purge,[]},
+     {add_module,emqx_exclusive_subscription},
```

Comment:
It is better place it before these changed modules?  
And,  do we need to create the mnesia table in hot-upgrading process?